lisa/tools/mdadm.py

```diff
@@ -47,3 +44,92 @@ def stop_raid(
         volume_name: str = "/dev/md0",
     ) -> None:
         self.run(f"--stop {volume_name}", force_run=True, sudo=True)
+
+    @classmethod
+    def _windows_tool(cls) -> Optional[Type[Tool]]:
+        return WindowsMdadm
+
+    def _install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        posix_os.install_packages("mdadm")
+        return self._check_exists()
+
+
+class WindowsMdadm(Mdadm):
+    @property
+    def command(self) -> str:
+        return "powershell"
+
+    @property
+    def can_install(self) -> bool:
```

Comment:
It shouldn't be installable, right? Override _check_exists instead of the can_install. That make the logic meaningful.